### ISSUE TITLE
Docs: Navigation Links in wrong place & wrong name 

### ISSUE THREAD

**Issue by torkelo** [grafana/grafana#23919]
In the application we refer to this as Dashboard links not navigation links, we should align this (in app or docs) 

Also this article is in the wrong location, should be under dashboard features, not directly under features.

<img width="578" alt="Screen Shot 2020-04-27 at 09 46 10" src="https://user-images.githubusercontent.com/10999/80347057-11851000-886c-11ea-86b3-01f0efe1c8ec.png">

**Comment by oddlittlebird** [comment]
If this was just about dashboard links, then I would agree with you. However, this topic discusses panel links and data links as well.

When I converted @marcusolsson 's [blog post](https://grafana.com/blog/2019/12/10/pro-tips-dashboard-navigation-using-links/) to a topic, I considered breaking it up into three topics: dashboard links, panel links, and data links. I opted to keep them together in one topic, because they are all really "link to visualizations." 

Because of this, they would not fit under Dashboard features or under Panels. As I continue to refine and update the documentation, we will re-evaluate where this content best fits and how best to organize it.
